examples/experimental/dagster-airlift/dagster_airlift/mwaa/auth.py

```diff
@@ -31,16 +32,17 @@ def get_session_info(region: str, env_name: str) -> Tuple[str, str]:
 
 
 class MwaaSessionAuthBackend(AirflowAuthBackend):
-    def __init__(self, region: str, env_name: str):
+    def __init__(self, region: str, env_name: str, profile_name: Optional[str] = None):
```

Comment:
Hmmm maybe we should just take in the `boto.Session` object directly as we might just replicate the entire interface here.

Reply:
I think we should just revisit the interface in general. I think we should be aiming to only auth the mwaa session once in general, not just retrieving the client I think.

Reply:
kk